### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_WebAppRegister_NonTextContrast: The Color contrast ratio of the focus indicator of the "Email", "Password", "Confirm password" and "Register" button is less than 3:1.

### ISSUE THREAD

**Issue by Swati1700** [dotnet/aspnetcore#42395]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopWeb;#Win11;#ColorContrast;#Benchmark;#Chrome;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2
Chrome: Version 102.0.5005.115 (Official Build) (64-bit)

### Steps to Reproduce: 

1. open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir WebApp"
3. move to the created directory using command "cd foldername"
4. run the following command "dotnet new webapp -au Individual" and hit enter.
5. run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. browser page will open with above template loaded
9. Test the Register page
10. Turn on Color contrast analyzer and check that The Color contrast ratio of focus indicator for "Email", "Password" and "Register" is less than 3:1.

### Actual:
The Color contrast ratio of the focus indicator of the "Email", "Password", "Confirm password" and "Register" button is less than 3:1.

**Similar issue also observed in the below scenarios:** 
**Issue 1: ASP.NET Core - WebApp Login**

1. open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir WebApp"
3. move to the created directory using command "cd foldername"
4. run the following command "dotnet new webapp -au Individual" and hit enter.
5. run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. browser page will open with above template loaded
9. once done with registration
10. Test Login page
11. Turn on Color contrast analyzer and check that The Color contrast ratio of focus indicator for "Email", "Password", "Remember me?" and "Log in" button is less than 3:1.

**Issue2: ASP.NET Core - WebApp Forgot Password**

1. open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir WebApp"
3. move to the created directory using command "cd foldername"
4. Run the following command "dotnet new webapp -au Individual" and hit enter.
5. run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. browser page will open with above template loaded
9. once done with registration
10. After Login with registered account
11. Tab and navigate to "forgot your password?" link and hit enter. 
12. Turn on Color contrast analyzer and check that The Color contrast ratio of focus indicator for "Email", "Reset Password" button is less than 3:1.

### Expected:
The Color contrast ratio of the focus indicator of the "Email", "Password", "Confirm password" and "Register" button should be more than or equal to 3:1.

### User Impact:
Low vision users won't be able to see the focus indicator.
![42395_A11y_ NET Core_ASP NETCore_WebAppRegister_NonTextContrast](https://user-images.githubusercontent.com/86710888/175489607-a5c0fa91-6a2a-4fb8-9bd5-27caf15b04f9.png)

**Comment by HaoK** [comment]
Should be fixed by https://github.com/dotnet/aspnetcore/pull/43177

**Comment by Swati1700** [comment]
We downloaded the latest RC1 build from the first column of the above table and verified the bug, and the bug still reproduces.
Please find the attachment of the version.
![image](https://user-images.githubusercontent.com/86710888/185587091-32f081a1-51bd-4f0d-99c1-f76e8aa98de9.png)

Could you please provide the repro steps that you followed, if the issue is not reproducing at your end.

**Comment by merriemcgaw** [comment]
@Swati1700 - latest builds should now have it. There was a small issue with flowing into the installers, but it should be addressed now.

**Comment by Swati1700** [comment]
We have installed latest build dotnet-sdk-7.0.100-rc.2.22452.3-win-x64 but getting a below error.
![image](https://user-images.githubusercontent.com/86710888/188159657-095b5474-426e-4874-b607-971bc2620ad4.png)

**Comment by merriemcgaw** [comment]
@Swati1700 - the updated machine setup instructions are as follows:
1. On a clean machine install .NET RC2 SDK from the **Top, Middle** cell in the [Installer Table](https://github.com/dotnet/installer#table)
2. Open a command prompt and enter the following command: dotnet nuget add source -n dotnet7 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json
3. Follow the test repro steps from here.

**Comment by Swati1700** [comment]
Verified the issue and the issue is fixed. Hence closing the bug.
![image](https://user-images.githubusercontent.com/86710888/192514406-c1c53e8d-bd7f-436e-b29c-7eb09bd4201f.png)
